packages/documentation-site/patternfly-docs/content/design-guidelines/styles/typography/typography.js

```diff
@@ -23,172 +22,138 @@ export const TitleLevel = ({ asGrid, styleProps, children, title = '', note = ''
+    fontFamilyText: "Red Hat Display",
     textStyle: "4xl (RHD md 36)"
   },
   hero: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (medium)",
-    // variableName: "--pf-v6-global--FontSize--3xl",
+    variableName: "--pf-t--global--font--size--3xl",
     fontSize: "28px",
     lineHeight: "1.3",
-    fontFamily: "RedHatDisplay",
+    fontFamily: "--pf-t--global--font--family--200",
+    fontFamilyText: "Red Hat Display",
     textStyle: "3xl (RHD md 28)"
   },
   first: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (medium)",
-    // variableName: "--pf-v6-global--FontSize--2xl",
+    variableName: "--pf-t--global--font--size--2xl",
     fontSize: "24px",
     lineHeight: "1.3",
-    fontFamily: "RedHatDisplay",
+    fontFamily: "--pf-t--global--font--family--200",
+    fontFamilyText: "Red Hat Display",
     textStyle: "2xl (RHD md 24)"
   },
   second: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (medium)",
     fontSize: "20px",
-    // variableName: "--pf-v6-global--FontSize--xl",
+    variableName: "--pf-t--global--font--size--xl",
     lineHeight: "1.5",
-    fontFamily: "RedHatDisplay",
+    fontFamily: "--pf-t--global--font--family--200",
+    fontFamilyText: "Red Hat Display",
     textStyle: "xl (RHD md 20)"
   },
   third: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (medium)",
     fontSize: "18px",
-    // variableName: "--pf-v6-global--FontSize--lg",
+    variableName: "--pf-t--global--font--size--lg",
     lineHeight: "1.5",
-    fontFamily: "RedHatDisplay",
+    fontFamily: "--pf-t--global--font--family--200",
+    fontFamilyText: "Red Hat Display",
     textStyle: "lg (RHD md 18)"
   },
   fourth: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (medium)",
     fontSize: "16px",
-    // variableName: "--pf-v6-global--FontSize--md",
+    variableName: "--pf-t--global--font--size--md",
     lineHeight: "1.5",
-    fontFamily: "RedHatDisplay",
+    fontFamily: "--pf-t--global--font--family--200",
+    fontFamilyText: "Red Hat Display",
     textStyle: "md (RHD md 16)"
   },
   body: {
-    fontWeight: "400",
-    fontWeightText: "400 (regular)",
-    fontSize: "16px",
-    // variableName: "--pf-v6-global--FontSize--md",
-    lineHeight: "1.5",
-    fontFamily: "RedHatText",
-    textStyle: "md (RHT rg 16)"
-  },
-  small: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (regular)",
     fontSize: "14px",
-    // variableName: "--pf-v6-global--FontSize--sm",
+    variableName: "--pf-t--global--font--size--body--default",
     lineHeight: "1.5",
-    fontFamily: "RedHatText",
-    textStyle: "sm (RHT rg 14)"
+    fontFamily: "--pf-t--global--font--family--100",
```

Comment:
And all the `--pf-t--global--font--family--100` can probably be `--pf-t--global--font--family--body`